src/DotNetEd.CoreAdmin/Views/CoreAdminData/Index.cshtml

```diff
@@ -61,7 +62,27 @@
                         var base64 = ImageUtils.WebBase64EncodeImageByteArrayOrNull(lambda.Compile().Invoke(value));
                         return base64 == null ? string.Empty : $"<img height='48' src='{base64}'>";
                     } ).Encoded(false);
-            }   
+            }
+            else if (entityProperty.PropertyType.IsAssignableTo(typeof(IEnumerable)) && entityProperty.PropertyType != typeof(string))
+            {
+                var lambda = Expression.Lambda<Func<object, object>>(property, entity);
+
+                columns.Add(lambda).Titled(entityProperty.Name)
+                    .RenderedAs((value) =>
+                    {
+                        var finalString = "<ul>";
+                        try
+                        {
+                            foreach (var raw in (IEnumerable)entityProperty.GetValue(value)!)
+                            {
+                                finalString += $"<li><a href=\"{Url.Action("Index", new { id = raw.GetType().Name })}\">{raw}</a></li>";
```

Comment:
strings behave like value variables and each time you add to them create another variable and its not optimum, better to use stringbuilder